apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -28,10 +28,14 @@ import React, { useEffect } from 'react'
 const SORTABLE_PROPERTIES = ['advancedAt', 'amount']
 const PAYMENTS_DEFAULT_SORT_BY = ['advancedAt_DESC']
 const DEFAULT_DATE_RANGE: [Dayjs, Dayjs] = [dayjs().subtract(1, 'week'), dayjs()]
+const DEFAULT_DATE_RANGE_STR: [string, string] = [String(DEFAULT_DATE_RANGE[0]), String(DEFAULT_DATE_RANGE[1])]
 
 const ROW_GUTTER: [Gutter, Gutter] = [0, 40]
 const TAP_BAR_ROW_GUTTER: [Gutter, Gutter] = [0, 20]
 
+let isDefaultFilterApplied = false
```

Comment:
IS_DEFAULT_FILTER_APPLIED
SHOULD_APPLY_DEFAULT_FILTER

constants shouldn't look like variables for readability sake. :)

Reply:
These variables are really variables.